### PR TITLE
Update branding to 3.1.15

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Update to ubuntu 18.04

### CODE REVIEW

**Review by wtgodbe — APPROVED** [review]

**Comment by JunTaoLuo** [comment]
Looks like updating to ubuntu 18.04 breaks a test. I'll take a look and see if it's something obvious. Otherwise, I'll revert the unbuntu update to unblock the branding.

**Comment by dougbu** [comment]
> Looks like updating to ubuntu 18.04 breaks a test

Hold off on this update for now.

**Comment by JunTaoLuo** [comment]
Interesting, I fixed the assembly load issue but it seems like the core logic at https://github.com/dotnet/extensions/blob/release/3.1/src/Hosting/Systemd/src/SystemdHelpers.cs#L28 is broken on newer ubuntu versions. I'll file an issue since this is potentially a product issue and undo the ubuntu update.

**Pull request merged**
